static/js/components/sidenav.vue

```diff
@@ -320,4 +323,10 @@
   .request li ~ ul.nav {
     display: none;
   }
+  hr {
```

Comment:
what is this for?

Reply:
There is a subtle horizontal line under the buttons in the sidenav.